### PR TITLE
update FT and NFT test cases

### PR DESCRIPTION
**Pull request by githubckgoh1439**
update for FT and NFT test cases

### CODE REVIEW

File: tests/non_fungible_test.go [hunk 1]
```diff
@@ -96,9 +102,11 @@ func makeNonFungibleTokenTxs() []*testCase {
...
-		{"nonFungibleToken", false, false, "Update NFT endorserlist [TNFT-validate-update-endorserlist-001]- Allowed for the empty endorser-list. commit", "acc-40", "400000000cin", 0, NonFungibleTokenInfo{"update-nft-endorserlist", "", "", "TestNonfungibleToken-validate-update-endorserlist-001", "TNFT-validate-update-endorserlist-001", "", "", "", "", "item-properties", "item-metadata", false, false, true, true, false, "nft-jeansoon", "0", "nft-carlo", "default", "", "4", "10000000", []string{}}, "MEMO: with empty endorser-list", nil}, // empty endorser-list
-		{"nonFungibleToken", false, false, "Update NFT endorserlist [TNFT-validate-update-endorserlist-001]- Happy Path. commit", "acc-40", "400000000cin", 0, NonFungibleTokenInfo{"update-nft-endorserlist", "", "", "TestNonfungibleToken-validate-update-endorserlist-001", "TNFT-validate-update-endorserlist-001", "", "", "", "", "item-properties", "item-metadata", false, false, true, true, false, "nft-jeansoon", "0", "nft-carlo", "default", "", "4", "10000000", []string{"nft-jeansoon", "nft-carlo"}}, "MEMO: tx passed", nil},              // make use this 'MEMO: tx passed' to compile the unique tx-hash
+		///goh20200818
```
Comment (yenkhoon) [inline tests/non_fungible_test.go:105]: please remove

File: tests/non_fungible_test.go [hunk 2]
```diff
@@ -350,6 +358,13 @@ func makeNonFungibleTokenTxs() []*testCase {
 		{"nonFungibleToken", true, true, "Freeze non fungible token - Invalid Token symbol - TNFT-111", "mostafa", "100000000cin", 0, NonFungibleTokenInfo{"freeze", "10000000", "nft-mostafa", "TestNonFungibleToken-111", "TNFT-111", "", "", "token metadata", "", "", "", true, false, false, false, false, "nft-jeansoon", "0", "nft-carlo", "", "", "", "", []string{""}}, "", nil},
 		{"nonFungibleToken", true, true, "Re-Freeze non fungible token - Error due to Token is frozen.", "nft-mostafa", "0cin", 0, NonFungibleTokenInfo{"freeze", "", "", "", "TNFT", "", "", "token metadata", "", "", "", true, false, false, true, false, "nft-jeansoon", "0", "nft-carlo", "", "", "", "", []string{""}}, "", nil}, // [Re-action]
 
+		//-D
```
Comment (yenkhoon) [inline tests/non_fungible_test.go:361]: please remove

File: tests/fungible_test.go [hunk 3]
```diff
@@ -36,6 +36,40 @@ func makeFungibleTokenTxs() []*testCase {
 
 	tcs := []*testCase{
 
+		//-D
+		///2. 20200818-MsgMintFungibleToken
+		{"token", false, false, "Create token TFT-772 - Dynamic-supply with mint unlimited", "acc-40", "100000000cin", 0, TokenInfo{"create", "100000", "mostafa", "TestToken-772", "TFT-772", 8, "acc-40", "", "", false, "0", false, false, false, "", "", "", "", "", ""}, "", nil},
+		{"token", false, false, "Approve token TFT-772 - for Dynamic-supply with mint unlimited purpose.", "token-auth-1", "0cin", 0, TokenInfo{"approve", "", "", "TestToken-772", "TFT-772", 0, "", "", "", false, "0", false, false, true, "0", "token-prov-1", "0", "token-issuer-1", "default", ""}, "", nil},
+		{"token", true, true, "Mint token TFT-772 - Error, due to Invalid token symbol field length which maximum is 40.", "acc-40", "100000000cin", 0, TokenInfo{"mint", "", "", "TestToken-772-name", "TFT-772-aaaaabbbbbcccccdddddeeeeefffffggggghhhhhiiiiijjjjjkkkkklllllmmmmmnnnnnooooopppppqqqqqrrrrrssssstttttuuuuuvvvvvwwwwwxxxxxyyyyyzzzzzz11111222223333344444555556666677777888889999900000", 8, "acc-40", "carlo", "", false, "0", false, false, true, "0", "", "", "", "", ""}, "", nil},
+
+		//-D
```
Comment (yenkhoon) [inline tests/fungible_test.go:45]: remove please

File: tests/fungible_test.go [hunk 4]
```diff
@@ -36,6 +36,40 @@ func makeFungibleTokenTxs() []*testCase {
...
+		{"token", false, false, "Approve token TFT-773 - for Dynamic-supply with mint unlimited purpose.", "token-auth-1", "0cin", 0, TokenInfo{"approve", "", "", "TestToken-773", "TFT-773", 0, "", "", "", false, "0", false, false, true, "0", "token-prov-1", "0", "token-issuer-1", "default", ""}, "", nil},
+		{"token", false, false, "Mint token TFT-773 - Dynamic-supply which allowed to mint-token with 0 amount.", "acc-40", "100000000cin", 0, TokenInfo{"mint", "", "", "TestToken-773-name", "TFT-773", 8, "acc-40", "carlo", "", false, "0", false, false, true, "0", "", "", "", "", ""}, "", nil},
+		{"token", true, true, "Transfer token TFT-773 -Error due to Invalid token symbol field length which maximum is 40.", "acc-40", "100000000cin", 0, TokenInfo{"transfer", "", "", "", "TFT-773-aaaaabbbbbcccccdddddeeeeefffffggggghhhhhiiiiijjjjjkkkkklllllmmmmmnnnnnooooopppppqqqqqrrrrrssssstttttuuuuuvvvvvwwwwwxxxxxyyyyyzzzzzz11111222223333344444555556666677777888889999900000", 8, "acc-40", "eve", "", false, "", true, false, false, "0", "", "", "", "", ""}, "", nil},
+
+		//-D
```
Comment (yenkhoon) [inline tests/fungible_test.go:52]: remove all -D

File: tests/non_fungible_test.go (hunk 1 above)
Comment (githubckgoh1439) [reply tests/non_fungible_test.go]: manual remove

File: tests/non_fungible_test.go (hunk 2 above)
Comment (githubckgoh1439) [reply tests/non_fungible_test.go]: manual remove

File: tests/fungible_test.go (hunk 4 above)
Comment (githubckgoh1439) [reply tests/fungible_test.go]: manual remove

**Pull request merged**